File: 107598019/HW2/Software Architecture_XML/Software Architecture/usecase/addCourse/AddCourseOutput.cs
using Software_Architecture.adapter;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Software_Architecture.usecase.addCourse
{
    public interface AddCourseOutput
    {
        void setTitle(string title);

        string getTitle();

        ViewModel getViewModel();
    }
}

File: 107598042/hw1/src/main/java/UseCase/AddCourse/AddCourseOutput.java
package UseCase.AddCourse;

import UseCase.CourseDTO;
import UseCase.Output;

public class AddCourseOutput implements Output {
    private CourseDTO courseDTO;

    public AddCourseOutput() {

    }

    public void setCourseDTO(CourseDTO courseDTO) {
        this.courseDTO = courseDTO;
    }

    public CourseDTO getCourseDTO() {
        return courseDTO;
    }
}
